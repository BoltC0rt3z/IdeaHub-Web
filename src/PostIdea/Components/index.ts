import Footer, { MobileFooter } from './Footer';

export { Footer, MobileFooter };
