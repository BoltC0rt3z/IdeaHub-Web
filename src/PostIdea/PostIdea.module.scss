@import '../styles/styles.module.scss';

.postIdea {
  flex-direction: column;
  background: $color-light;

  .profile {
    padding-left: 8px;
  }

  :global(.ant-input:placeholder-shown) {
    font-size: 20px;
  }
}

.input {
  cursor: pointer;
}
