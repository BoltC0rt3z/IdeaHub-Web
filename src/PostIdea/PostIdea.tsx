import React from 'react';
import { Row } from 'antd';
import { InputProps } from 'antd/lib/input';

import { BorderLessInput } from '../Components';
import styles from './PostIdea.module.scss';

const PostIdea: React.FC<InputProps> = (props) => {
  return (
    <Row type="flex" className={styles.postIdea}>
      <Row className={styles.profile}>Brian Ogutu</Row>
      <Row>
        <BorderLessInput {...props} className={styles.input} />
      </Row>
    </Row>
  );
};

export default PostIdea;
