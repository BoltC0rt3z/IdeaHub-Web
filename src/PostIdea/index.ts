import PostIdea from './PostIdea';

export { PostIdea };
