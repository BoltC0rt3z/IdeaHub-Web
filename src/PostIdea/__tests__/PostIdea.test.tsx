import React from 'react';
import { mount, ReactWrapper } from 'enzyme';
import PostIdea from '../PostIdea';

describe('Tests for posting idea container', () => {
  const wrapper: ReactWrapper = mount(<PostIdea />);

  it('Should render PostIdea component without crushing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
