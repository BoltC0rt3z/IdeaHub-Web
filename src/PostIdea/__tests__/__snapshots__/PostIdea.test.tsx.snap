// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for posting idea container Should render PostIdea component without crushing 1`] = `ReactWrapper {}`;
